Sources/ApolloCodegenTestSupport/MockFileManager.swift

```diff
@@ -28,10 +28,31 @@ public class MockFileManager: FileManager {
   private var closures: [String: Closure] = [:]
   private var closuresToBeCalled: Set<String> = []
 
+  /// If `true` then all called closures must be mocked otherwise the call will fail. When `false` any called closure
+  /// that is not mocked will fall through to `super`. As a byproduct of `false`, all mocked closures must be called otherwise
+  /// the test will fail.
+  let strict: Bool
```

Comment:
The previous behaviour was always strict, it's now strict be default. Turning this into a property allows for less mocking code and cleaner tests. There is the added 'safety net' in `deinit` when strict is false.